### PR TITLE
update todo

### PR DESCRIPTION
**Pull request by markus2330**
Tasks for August, please review if everything is clear.

### CODE REVIEW

**Review by horenso — APPROVED** [review]

File: TODO.md [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
-dump all registers
+## Weather station
 
-note when raising `> 23°` for closing
-warn when raising `> 30°` and no wind
-warn when raising `> 35°`
-(and remove warning again if `< 20°`)
+Assignee: Felix
+
+implement all error codes & send i18n errors into Nextcloud ping chat
```
Comment (fel115) [inline TODO.md:13]: what is meant by i18n? The clima sensor only has two error codes.

File: TODO.md [hunk 2]
```diff
@@ -29,6 +32,10 @@ lights on/off
 
 ## Env
 
+Assignee: Felix
+
+create use case: warn/alarm on in-door temperatures
```
Comment (fel115) [inline TODO.md:37]: should the warning temperature be set by the user?

Comment (markus2330) [reply TODO.md:37]: Not necessarily, we can try if everyone is happy with 50° and 28°. Once people disagree, we can still make it a configuration option.

Comment (markus2330) [reply TODO.md:37]: In general it is quite easy to replace constants to configuration settings. But the other way round is not problematic (you never know if users might use it).

File: TODO.md (hunk 1 above)
Comment (fel115) [reply TODO.md:13]: Ohh Yeah, right I already did it :sweat_smile:

Comment (markus2330) [reply TODO.md:13]: Please remove this part then in #49.

**Pull request merged**